libs/language-server/src/lib/constraint/length-constraint-meta-inf.ts

```diff
@@ -59,6 +59,19 @@ export class LengthConstraintMetaInformation extends ConstraintMetaInformation {
         }
       },
     );
+    super.docs = {
+      description:
+        'Limits the length of a string with an upper and/or lower boundary. Only values with a length within the given range are valid.',
+      examples: [
+        {
+          description: 'A string with 0 to 2147483647 characters.',
```

Comment:
Is there a reason for picking that upper boundary? I would have used something smaller, e.g. `12`.

Reply:
Yes, it is the official largest string length of Java.